phpBB/assets/javascript/installer.js

```diff
@@ -21,6 +21,26 @@
 	// Intercept form submits
 	interceptFormSubmit($('#install_install'));
 
+	// Interception of click on the navigation link
+	$('.my_send').click( function (e) {
```

Comment:
I'd recommend using a more meaningful class name that is based on it's purpose. `my_send` actually looks like you're trying to use hungarian notation which is something we do not approve of.